app/controller.js

```diff
@@ -40,10 +40,27 @@ exports.getArticles = (req,res,next) =>{
     })
 }
 
-exports.getAllArticles = (req,res,nest) =>{
+exports.getAllArticles = (req,res,next) =>{
 
     selectAllArticles().then(({rows})=>{
 
         return res.status(200).send({articles:rows})
     })
+}
+
+exports.getComments = (req,res,next) =>{
+
+    const articleId = req.params.article_id
+
+    selectSpecifiedComments(articleId).then(({rows})=>{
+        const comments = rows
+ 
+        if(rows.length===0){return res.status(404).send({msg:"No comments!!"})}
+      
+        return res.status(200).send({comments:rows})
+
+    }).catch(({code})=>{if(code==="22P02"){
```

Comment:
this should be handled in your error handling middleware, you don't need to be concerned about error codes in your controller, you should just be passing them to the middleware to deal with it